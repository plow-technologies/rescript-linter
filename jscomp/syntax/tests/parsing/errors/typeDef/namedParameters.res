type  draw = (stroke: pencil) => unit
