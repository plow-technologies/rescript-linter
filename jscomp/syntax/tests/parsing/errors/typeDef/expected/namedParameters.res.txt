
  Syntax error!
  tests/parsing/errors/typeDef/namedParameters.res:1:15-20

  1 â”‚ type  draw = (stroke: pencil) => unit
  2 â”‚ 

  A labeled parameter starts with a `~`. Did you mean: `~stroke`?

type nonrec draw = stroke:pencil -> unit